zebrad/tests/common/sync.rs

```diff
@@ -74,7 +74,7 @@ pub const FINISH_PARTIAL_SYNC_TIMEOUT: Duration = Duration::from_secs(11 * 60 *
 
```

Comment:
Do we want to increase `FINISH_PARTIAL_SYNC_TIMEOUT` too?

Reply:
The partial sync uses cached state, and it is not timing out or failing, so it should be fine.